lib/tasks/assets.rake

```diff
@@ -1,33 +1,8 @@
+# frozen_string_literal: true
+
 # Tasks taken from Gitlab's yarn.rake and assets.rake files.
 
 namespace :portus do
-  namespace :yarn do
```

Comment:
No need anymore?

Reply:
We don't use it anywhere, and I don't think it's that useful.